File: client-app/src/app/models/comment.ts
import { Interface } from "readline";
import { LearningUnit } from "./learningUnit";
import { User } from "./user";

export interface LUComment {
    id?: string;
    user?: User | null;
    learningUnitId?: string;
    learningUnit?: LearningUnit;
    content?: string;
    createdAt?: Date;
}

File: Domain/Comment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain
{
    public class Comment
    {
        public Guid Id { get; set; }
        public AppUser? User { get; set; }
        public Guid LearningUnitId { get; set; }
        public LearningUnit? LearningUnit { get; set; }
        public string Content { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}
